nrt/data/DynaFlow/PhaseShifterPSign/PhaseShifterPSign.iidm

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="ISO-8859-1" standalone="no"?>
+<!--
+    Copyright (c) 2023, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source suite of
+    simulation tools for power systems.
+-->
+<network xmlns="http://www.itesla_project.eu/schema/iidm/1_0" id="ieee14bus" caseDate="2017-06-09T10:14:24.146+02:00" forecastDistance="0" sourceFormat="CIM1">
+  <substation id="SubStation" name="SubStation">
+    <voltageLevel id="VoltageLevel1" name="VoltageLevel1" nominalV="400" topologyKind="BUS_BREAKER">
+      <busBreakerTopology>
+        <bus id="Bus1" v="400" angle="0"/>
+      </busBreakerTopology>
+      <generator id="Generator" name="Generator" energySource="NUCLEAR" minP="-9999" maxP="9999" voltageRegulatorOn="true" targetP="0" targetQ="0" targetV="400" bus="Bus1" connectableBus="Bus1" p="0" q="0">
+        <minMaxReactiveLimits minQ="-999" maxQ="999"/>
+      </generator>
+    </voltageLevel>
+    <voltageLevel id="VoltageLevel2" name="VoltageLevel2" nominalV="400" topologyKind="BUS_BREAKER">
+      <busBreakerTopology>
+        <bus id="Bus2" v="400" angle="0"/>
+      </busBreakerTopology>
+      <load id="Load" name="Load" loadType="UNDEFINED" p0="0" q0="0" bus="Bus2" connectableBus="Bus2" p="0" q="0"/>
+    </voltageLevel>
+    <twoWindingsTransformer id="PhaseShifter" name="PhaseShifter" r="0" x="10" g="0" b="0" ratedU1="400" ratedU2="400" voltageLevelId1="VoltageLevel1" bus1="Bus1" connectableBus1="Bus1" voltageLevelId2="VoltageLevel2" bus2="Bus2" connectableBus2="Bus2" p1="0" q1="0" p2="0" q2="0">
+      <phaseTapChanger lowTapPosition="1" tapPosition="13" regulationMode="CURRENT_LIMITER" regulationValue="770" regulating="true">
```

Comment:
What we can do is change CURRENT_LIMITER to ACTIVE_POWER_CONTROL, change the regulationValue (i.e the value for PTarget multiplied by 100 to have it in MW)